src/app/organization/manage-funds/manage-funds.component.ts

```diff
@@ -0,0 +1,24 @@
+/** Angular Imports */
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MatTableDataSource, MatPaginator, MatSort } from '@angular/material';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
```

Comment:
Add relevant component, see other components such as system>codes

Reply:
do you mean comments?

Reply:
Yes, I apologise

Reply:
@punwai its okay :) I have made the changes.